pkg/ruptela/vehicle-convert-funcs_gen.go

```diff
@@ -103,6 +103,12 @@ func ToLowVoltageBatteryCurrentVoltage0(originalDoc []byte, val string) (float64
 	return mV / 1000, nil
 }
 
+// ToOBDDTCList0 converts data from field 'signals.93' of type string to 'Vehicle.OBD.DTCList' of type string.
+// Vehicle.OBD.DTCList: List of currently active DTCs formatted according OBD II (SAE-J2012DA_201812) standard ([P|C|B|U]XXXXX )
+func ToOBDDTCList0(originalDoc []byte, val string) (string, error) {
+	return ignoreZero(Convert93(val))
```

Comment:
`ignoreZero` function returns a (float64, error)
so we can not use it here as the return for `ToOBDDTCList0` since this function is meant to return (string, error)
We need to determine how to convert the hex value from the ruptela device to a list DTCs